lib/polyn/templates/tf/event_registry.tf

```diff
@@ -0,0 +1,3 @@
+resource "jetstream_kv_bucket" "POLYN_SCHEMAS" {
+  name = "POLYN_SCHEMAS"
```

Comment:
Do we want to add replicas to this? I feel like we'd want at least 3 in case of a failure.